doxygen/parallelism_support/opencl_support.md

```diff
@@ -94,20 +94,135 @@ Once you have done the above step, `runTests.py` should execute with the GPU ena
 
 ## Using the OpenCL backend
 
-We currently have support for the following methods
+The OpenCL backend can be used for reverse mode AD as well as primitive functions on containers of basic C++ scalar types. Below is the list of functions and distributions that are currently supported.
+
+### Reverse mode
+
+An example of using OpenCL supported Stan Math functions for reverse mode AD is shown below: 
+
+```cpp
+using stan::math::matrix_cl;
+using stan::math::var_value;
+using stan::math::to_matrix_cl;
+using stan::math::from_matrix_cl;
+
+Eigen::Matrix<double, -1, -1> A(N,M);
+Eigen::Matrix<double, -1, -1> B(M,N);
+// ... fill matrices A and B
+var_value<matrix_cl<double>> A_cl = to_matrix_cl(A);
+var_value<matrix_cl<double>> B_cl = to_matrix_cl(B);
+var_value<matrix_cl<double>> C_cl = A_cl * B_cl;
+var_value<Eigen::Matrix<double, -1, -1>> C = from_matrix_cl(C_cl);
+C(0,0).grad();
+```
+
+#### Supported functions
 
-- bernoulli_logit_glm
 - cholesky_decompose
-- categorical_logit_glm
 - gp_exp_quad_cov
 - mdivide_right_tri
 - mdivide_left_tri
 - multiplication
+- sum
+
+#### Supported distributions
```

Comment:
So far only lpdf/lpmf functions are supported (no cdf/lcdf/lccdf).

Reply:
Added a sentence that mentions that. Let me know if you would prefer to have the list explicitly mention the suffices like
```
- bernoulli_lpmf
- bernoulli_logit_lpmf
- bernoulli_logit_glm_lpmf
- beta_lpdf
...
```

Reply:
I expect I will start adding  cdf/lcdf/lccdf within a month or two. We will need to mention explicit suffixes at that point.

Reply:
Ok, explicit suffices it is! :)